eng/pipelines/coreclr/templates/build-perf-maui-apps.yml

```diff
@@ -178,11 +178,12 @@ steps:
       cp MauiTesting.csproj MauiTesting.csproj.bak
       sed -i'' -e 's/net6.0-ios;net6.0-maccatalyst/net6.0-ios/g' MauiTesting.csproj
 
-      ../dotnet publish -bl:MauiiOS.binlog -f net6.0-ios --self-contained -r ios-arm64 -c Release /p:_RequireCodeSigning=false
+      ../dotnet publish -bl:MauiiOS.binlog -f net6.0-ios --self-contained -r ios-arm64 -c Release /p:_RequireCodeSigning=false /p:ApplicationId=net.dot.mauitesting
+      ls -aR
```

Comment:
Are these going to stay around, or were they just for testing?

Reply:
Testing the above mentioned ApplicationId and to add the podcast app